docs/src/references/c8y-configuration-management.md

```diff
@@ -89,16 +149,68 @@ files = [
   ]
 ```
 
+Along this `c8y_configuration_plugin` configuration for the main device,
+the configuration plugin expect a configuration file per child device
+that needs to be configured from the cloud.
+* The configuration for a child-device `$CHILD_DEVICE_ID`
+  is stored by default under `/etc/tedge/c8y/$CHILD_DEVICE_ID/c8y-configuration-plugin.toml`
+* These TOML files have the same schema as for the main device,
+  listing the configuration `files` and giving for each a `path` and possibly a `type`.
+* Note that the `path` doesn't need to be a file path.
+  It can be a key path in some registry of the child device or any name that makes sense for the child device.
+* As for the main device, the `type` is used to name the configuration file on the cloud.
+  All the configuration `type`s for a child devices are declared to the cloud on start
+  and on change of the `c8y/$CHILD_DEVICE_ID/c8y-configuration-plugin.toml` file.
+* The `user`, `group` and `mode` can be provided for a child-device configuration file
+  but will not be used by the main device if provided.   
+
+```shell
+$ ls /etc/tedge/c8y/*/c8y-configuration-plugin.toml
+/etc/tedge/c8y/child-1/c8y-configuration-plugin.toml 
+/etc/tedge/c8y/child-2/c8y-configuration-plugin.toml
+
+$ cat /etc/tedge/c8y/child-1/c8y-configuration-plugin.toml
+files = [
+    { path = '/var/camera.conf', type = 'camera' },
+    { path = '/var/sounds.conf', type = 'sounds' },
+  ]
+
+$ cat /etc/tedge/c8y/child-2/c8y-configuration-plugin.toml
+files = [
+    { path = '/var/ai/model' },
+  ]
+```
+
+On start and when one of these files is updated, the configuration plugin sends
+one [`119 SmartRest2 message`](https://cumulocity.com/guides/10.14.0/reference/smartrest-two/#119) per device
+to Cumulocity with the set of `type`s listed by the configuration
+(adding implicitly the `c8y-configuration-plugin` themselves).
+These messages can be observed over the MQTT bus of the thin-edge device.
+In the case of the example, 3 messages are sent - one for the main device and 2 for the child devices:
+
+```shell
+$ tedge mqtt sub 'c8y/s/ds/#'
+[c8y/s/ds] 119,c8y-configuration-plugin,tedge.toml,/etc/tedge/mosquitto-conf/c8y-bridge.conf,/etc/tedge/mosquitto-conf/tedge-mosquitto.conf,mosquitto
+[c8y/s/ds/child-1] 119,c8y-configuration-plugin,camera,sounds
+[c8y/s/ds/child-2] 119,c8y-configuration-plugin,/var/ai/model
```

Comment:
Same here. `c8y/s/us` should be correct.

Reply:
I will double-check.

Reply:
I was wrong => fixed